src/usecases/document/document.service.ts

```diff
@@ -157,6 +158,14 @@ export class DocumentService implements DocumentUseCase {
 		};
 	}
 
+	async getDocumentByAccountId({
+		accountId,
+	}: GetByAccountIdInput): Promise<DocumentEntity> {
+		return await this.documentRepository.getByAccountId({
+			accountId,
```

Comment:
Quando vc retorna uma promise direto, nao precisa de async/await
```suggestion
	getDocumentByAccountId({
		accountId,
	}: GetByAccountIdInput): Promise<DocumentEntity> {
		return this.documentRepository.getByAccountId({
			accountId,
```